tim1/src/main/java/isamrs/tim1/service/ReservationService.java

```diff
@@ -386,5 +392,56 @@ private void inviteFriendToFlight(String email, FlightReservationDTO flightRes,
 		return new ResponseEntity<MessageDTO>(
 				new MessageDTO("Successfully declined reservation.", ToasterType.SUCCESS.toString()), HttpStatus.OK);
 	}
+	
+	public MessageDTO createQuickFlightReservation(QuickFlightReservationDTO quickDTO) {
+		Flight f = flightRepository.findOneByFlightCode(quickDTO.getFlightCode());
+		if (f == null) {
+			return new MessageDTO("Flight does not exist.", ToasterType.ERROR.toString());
+		}
+		QuickFlightReservation qfr = new QuickFlightReservation();
+		double price = 0.0;
+		String[] idx = quickDTO.getSeat().split("_");
+		int row = Integer.parseInt(idx[0]);
+		int column = Integer.parseInt(idx[1]);
+		if (checkIfSeatIsReserved(f, row, column)) {
```

Comment:
Dates of reservations should also be checked
https://github.com/ISAMRS-TIM1/Projekat/blob/7a8defc8cf1d9e2b16b13cc2b97d776bab899128/tim1/src/main/java/isamrs/tim1/service/ReservationService.java#L273

Reply:
One seat can not be reserved for different dates.